test/nieSyntaxSpecs.js

```diff
@@ -0,0 +1,42 @@
+describe('Testing nie syntax services', function() {
+	var $injector;
```

Comment:
But I use it to load the syntaxServices...
$injector = angular.injector(['ng','app.services']);
syntaxServices = $injector.get('syntaxServices');
This is the way we did it in class.